app/modules/SilhouetteModule.scala

```diff
@@ -0,0 +1,161 @@
+package modules
+
+import com.google.inject.{ AbstractModule, Provides }
+import com.mohiva.play.silhouette.api.repositories.AuthInfoRepository
+import com.mohiva.play.silhouette.api.services._
+import com.mohiva.play.silhouette.api.util._
+import com.mohiva.play.silhouette.api.{ Environment, EventBus }
+import com.mohiva.play.silhouette.impl.authenticators._
+import com.mohiva.play.silhouette.impl.daos.{ CacheAuthenticatorDAO, DelegableAuthInfoDAO }
+import com.mohiva.play.silhouette.impl.providers._
+import com.mohiva.play.silhouette.impl.providers.oauth1._
+import com.mohiva.play.silhouette.impl.providers.oauth1.secrets.{ CookieSecretProvider, CookieSecretSettings }
+import com.mohiva.play.silhouette.impl.providers.oauth1.services.PlayOAuth1Service
+import com.mohiva.play.silhouette.impl.providers.oauth2._
+import com.mohiva.play.silhouette.impl.providers.oauth2.state.{ CookieStateProvider, CookieStateSettings, DummyStateProvider }
+import com.mohiva.play.silhouette.impl.providers.openid.services.PlayOpenIDService
+import com.mohiva.play.silhouette.impl.repositories.DelegableAuthInfoRepository
+import com.mohiva.play.silhouette.impl.services._
+import com.mohiva.play.silhouette.impl.util._
+import models.User
+import models.daos._
+import models.services.{ UserService, UserServiceImpl }
+import net.ceedubs.ficus.Ficus._
+import net.ceedubs.ficus.readers.ArbitraryTypeReader._
+import net.codingwell.scalaguice.ScalaModule
+import play.api.Configuration
+import play.api.libs.concurrent.Execution.Implicits._
+import play.api.libs.ws.WSClient
+
+/**
+ * The Guice module which wires all Silhouette dependencies.
+ *
+ * This implementation is based on https://github.com/sbrunk/play-silhouette-slick-seed
+ */
+class SilhouetteModule extends AbstractModule with ScalaModule {
+
+  /**
+   * Configures the module.
+   */
+  def configure() {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use : Unit = for procedures](https://www.codacy.com/app/HiP-App/HiPCMS/file/2283741794/issues/source?bid=3144114&fileBranchId=3144125#l40)